doc/topics/packaging/testing.rst

```diff
@@ -0,0 +1,150 @@
+.. _pkging-testing:
+
+================
+Testing packages
+================
+
+The package test suite
+======================
+
+The salt repo provides a test suite for testing basic functionality of our
+packages at ``<repo-root>/pkg/tests/``. You can run the install, upgrade, and
+downgrade tests. These tests run automatically on most PRs that are submitted
+against Salt.
+
+
+.. warning::
+
+    These tests make destructive changes to your system because they install the
+    built packages onto the system. They may also install older versions in the
+    case of upgrades or downgrades. To prevent destructive changes, run the
+    tests in an isolated system, preferably a virtual machine.
+
+Setup
+=====
+In order to run the package tests, the `relenv
+<https://github.com/saltstack/relative-environment-for-python>`_ onedir and
+built packages need to be placed in the correct locations.
+
+Place all salt packages for the applicable testing version in
+``<repo-root>/pkg/artifacts/``, and the onedir must be located under
+``<repo-root>/artifacts/``. Additionally, to ensure complete parity with Salt's
+CI/CD suite, place the ``nox`` virtual environment in
+``<repo-root>/.nox/test-pkgs-onedir`` The following are a few ways this can be
+accomplished easily.
```

Comment:
**Suggestion (non-blocking):** It just occurred to me reading this that this might be more clear in a bulleted list. Just a thought. Feel free to ignore!